.github/dependabot.yml

```diff
@@ -0,0 +1,41 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "cargo" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```

Comment:
Just double-checking, this means that dependabot is workspace-aware - right?

Reply:
Based on this: https://github.com/dependabot/dependabot-core/issues/1207#issuecomment-500635306 - I think it is.